cartography/models/googleworkspace/user.py

```diff
@@ -0,0 +1,105 @@
+from dataclasses import dataclass
+
+from cartography.models.core.common import PropertyRef
+from cartography.models.core.nodes import CartographyNodeProperties
+from cartography.models.core.nodes import CartographyNodeSchema
+from cartography.models.core.nodes import ExtraNodeLabels
+from cartography.models.core.relationships import CartographyRelProperties
+from cartography.models.core.relationships import CartographyRelSchema
+from cartography.models.core.relationships import LinkDirection
+from cartography.models.core.relationships import make_target_node_matcher
+from cartography.models.core.relationships import TargetNodeMatcher
+
+
+@dataclass(frozen=True)
+class GoogleWorkspaceUserNodeProperties(CartographyNodeProperties):
+    """
+    Google Workspace user node properties
+    """
+
+    id: PropertyRef = PropertyRef("id")
+    lastupdated: PropertyRef = PropertyRef("lastupdated", set_in_kwargs=True)
+
+    # User identifiers and basic info
```

Comment:
Not critical: Adding `department` and recovery email/phone will add tremendous value from IT/security perspective.